site/content/en/guides/consumer/compositepkgs/_index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Get and apply a remote composite package"
+linkTitle: "Composite Package"
+weight: 7
+type: docs
+description: >
+  Fetch a composite package from a remote git repository and apply its contents to
+  a cluster
+---
+
+{{% pageinfo color="warning" %}}
+
+#### Notice: Composite packages feature support is in alpha phase
+
+{{% /pageinfo %}}
+
+This guide walks you through an example to get, view, set and apply contents of a
+kpt package which has subpackages in it. A kpt package is a directory of
+resource configs with a valid `Kptfile` in it. A composite package is a `kpt` package
+with 1 or more subpackages in its directory tree.
+
+Principles:
+
+1. Each kpt package is an independent building block and
+   should contain resources(ex: setter definitions) of its own.
+2. If a package is present in the directory tree of parent package,
+   the configs of that package are out of scope for the actions performed
+   on the parent package.
+3. To run a command recursively on all the subpackages, users can leverage
+   `--recurse-subpackages(-R)` flag. This is equivalent to running the same
+   command on each package path in the directory tree.
+
+## Steps
+
+1. [Fetch a remote package](#fetch-a-remote-package)
+2. [View the package contents](#view-the-package-contents)
+3. [Set the setter parameters](#set-the-setter-parameters)
+4. [Apply the composite package](#apply-the-composite-package)
+
+## Fetch a remote package
+
+Fetch an example package which has subpackages using [kpt pkg get].
+
+### get command
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/hello-composite-pkg \
+hello-composite-pkg
+```
+
+## View the package contents
+
+The primary package artifacts are Kubernetes resource configuration
+(e.g. YAML files).
+
+### tree command
```

Comment:
Ditto. "tree command" is not a good section name. Instead focus on what user wants to do, not the command they're running. This should be in the form of.

1. Fetch the package
2. View contents
3. List setters
.....

The actual command name is implementation detail of the call to action.

Same for other guides being added.